### DISCUSSION TITLE
How to switch from Vite to Webpack

### DISCUSSION THREAD

**Discussion by kirschkern** [quasarframework/quasar#13731] [General - Components / Directives / etc]
I use the latest Quasar CLI. quasar v2.7.1 and @quasar/app-vite v1.0.1

After a couple of problems with Vite I think about switching the project to Webpack. Is this possible?
What do I need to change to use webpack? 
Or would it be better to setup a new Webpack based project and copy the source files?

**Comment by suleimanas** [comment]
Yes, it would be better to create new project and copy your source files

**Comment by alexrififi** [comment]
And what kind of problems with Vite? I switched to Vite from Webpack and am very pleased!

**Reply by kirschkern** [reply]
@medvinator Thanks for your support! 
Changes in my vue or scss code are sometimes not present with hot reload. At least more often than in my Webpack projects.
Changes in my separate js files are never included. I always need to restart the quasar dev server. This is the main reason for thinking about a switch. With Webpack these js file changes where only reflected after a page reload, but I didn't need to restart the server.

I'm pretty sure there must be a solution like adding the js folders to a vite or quasar config file, but I have no idea where to start and didn't found any solution after a couple of hours.
